test/main.sh

```diff
@@ -211,6 +211,11 @@ run_test() {
     fi
   fi
 
+  # Run the panic checker after every test. This ensures that the test fails if any panics occur (as long as DEBUG is set).
```

Comment:
So now setting `LXD_VERBOSE` or `LXD_DEBUG` can change the outcome of the tests, I don't think this is ideal but I also can't come up with an alternative.

Would you mind chipping in on this? @simondeziel

Reply:
I agree. I think it's possible to implement a recovery middleware for mux that can catch the panic before the net/http library does. This would allow us to log panics at error level. I didn't look into this in too much detail as I'm mainly adding this for #14324. The CI tests are all running with `LXD_VERBOSE`.